docs/wiki/framework/refuel-framework.md

```diff
@@ -95,3 +102,23 @@ The jerry can will now have a volume of 200 liters.
 ---| --------- | -----------
 0  |  `fuelTruck` | My fuel truck object
 1  |  `428` | New fuel supply
+
+### 2.4 Make an object into a refuel source
+
+`ace_refuel_fnc_makeSource`
+
+   | Arguments | Type | Optional (default value)
+---| --------- | ---- | ------------------------
+0  | Fuel Source | Object | Required
+1  | Amount (in liters) | Number | Optional (default: `0`)
+2  | Hooks positions | Array | Optional (default: `[[0,0,0]]`)
+**R** | None | None | Return value
+
+#### 2.4.1 Example
+
+`[cursorObject, 100] call ace_refuel_fnc_makeSource`
+
+   | Arguments | Explanation
+---| --------- | -----------
+0  |  `cursorObject` | Fuel source object
```

Comment:
One extra space on left in second column. Same in the next line.

Reply:
it's copy-paste